src/main/java/Main.java

```diff
@@ -390,6 +386,7 @@ private static String readStringNonBlocking(InputStream in, int maxToRead) throw
         in.read(buffer);
         return new String(buffer);
     }
+
     private static void trimOffOurOptions(List<String> arguments) {
         arguments.removeIf(arg -> arg.startsWith("--daemon") || arg.startsWith("--logfile") || arg.startsWith("--extractedFilesFolder")
```

Comment:
The original PR from @basil removed this filter, but maybe we should keep it to avoid the wrong option from being passed to the executable. WDYT @basil ? Can keep your behavior too, especially if we have upgrade guidelines

```suggestion
        arguments.removeIf(arg.startsWith("--logfile") || arg.startsWith("--extractedFilesFolder")
```